rpc/frmops.c

```diff
@@ -559,7 +559,7 @@ static void item_fail(struct m0_rpc_packet *p, struct m0_rpc_item *item, int rc)
 
         item->ri_error = rc;
         if (item->ri_error != 0) {
-                M0_LOG(M0_ERROR, "packet %p, item %p[%"PRIu32"] failed with"
+                M0_LOG(M0_DEBUG, "packet %p, item %p[%"PRIu32"] failed with"
```

Comment:
Why do you change the debug level?

Reply:
During ST testing, we observed that the logs were filled up with these error logs during the initial connection establishment due to asynchronous starting of the servers.
Reverted to M0_ERROR level.